pkg/kubelet/apis/well_known_labels.go

```diff
@@ -34,28 +34,20 @@ const (
 	// and GA labels to ensure backward compatibility.
 	// TODO: stop applying the beta Arch labels in Kubernetes 1.18.
 	LabelArch = "beta.kubernetes.io/arch"
-
-	// GA versions of the legacy beta labels.
-	// TODO: update kubelet and controllers to set both beta and GA labels, then export these constants
-	labelZoneFailureDomainGA = "failure-domain.kubernetes.io/zone"
-	labelZoneRegionGA        = "failure-domain.kubernetes.io/region"
-	labelInstanceTypeGA      = "kubernetes.io/instance-type"
```

Comment:
don't drop the instance-type label?

Reply:
Good catch, should be a part of https://github.com/kubernetes/kubernetes/pull/82049

Reply:
still outstanding?

Reply:
It was added back, wasn't caught as resolved in the diff though